impl/client/src/main/doc-js/pentaho/visual/base/spec/IViewEx.jsdoc

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2010 - 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `IViewEx` interface adds to the [IView]{@link pentaho.visual.base.spec.IView} interface
+ * convenience properties that can be specified when constructing a view instance.
+ *
+ * @name pentaho.visual.base.spec.IViewEx
+ * @interface
+ * @extends pentaho.visual.base.spec.IView
+ */
+
+/**
+ * The container DOM element where the view is rendered.
+ *
+ * @name domContainer
```

Comment:
The base interface `spec.IView` is not what you specify in the constructor but what you obtain from serialization. The `spec.IViewEx` adds to this interface additional arguments you can specify along with the serialization spec to initialize the view.